packages/ui/src/components/TextField/index.tsx

```diff
@@ -105,6 +116,10 @@ const StyledHelperTextBox = styled.div`
     margin-top: 14px;
   }
 `;
+const InputContainer = styled.div`
+  position: relative;
+  width: 100%;
+`;
 const StyledInput = styled.input<{ isAdmin: boolean; fontColor: string }>`
   width: 100%;
   padding: 8px 16px;
```

Comment:
```suggestion
const StyledInput = styled.input<{
  isAdmin: boolean;
  fontColor: string;
  isRight: boolean;
}>`
  width: 100%;
  // 아이콘 24x24 가정
  padding: ${({ isRight }) => (isRight ? '8px 50px 8px 16px' : '8px 16px')};

```

<img width="329" alt="image" src="https://github.com/CEOS-Developers/CEOS-FE/assets/65931227/4f4611cf-db5b-4611-88b5-74571d23a42f">

요런 느낌으로 가면 좋을 것 같아용